generators/client/templates/vue/src/main/webapp/app/admin/user-management/user-management.component.ts.ejs

```diff
@@ -69,6 +76,10 @@ export default class <%=jhiPrefixCapitalized%>UserManagementComponent extends mi
     }
   }
 
+  public handleSyncList(): void {
```

Comment:
@FabienEssid : you need to move this function just before line 63, as it is needed for Cassandra application.

Reply:
Oh you right ! I will fix it in few minutes